lib/jasmine-reportportal-reporter.js

```diff
@@ -52,23 +56,57 @@ class ReportportalReporter {
     }
 
     setDescription(description) {
-        this.additionalCustomParams = Object.assign(this.additionalCustomParams, description && { description: description.text });
+        const { suiteDescription } = this.suiteAdditionalCustomParams;
+
+        if (description && description.text.suite) {
+            this.suiteAdditionalCustomParams.suiteDescription = suiteDescription.concat(description.text);
+        } else {
+            this.additionalCustomParams = Object.assign(this.additionalCustomParams, description && { description: description.text });
+        }
     }
 
     registerListeners () {
         process.on(EVENTS.ADD_ATTRIBUTES, this.addAttributes.bind(this));
         process.on(EVENTS.ADD_DESCRIPTION, this.setDescription.bind(this));
     }
 
+    getSuiteAttributesBySuite (suite) {
+        const { suiteAttributes } = this.suiteAdditionalCustomParams;
+
+        return suiteAttributes && suiteAttributes.filter(attr => attr.suite === suite).map(attr => ({
+            key: attr.key,
+            value: attr.value,
+        }));
+    }
+
+    getSuiteDescriptionBySuite (suite) {
+        const { suiteDescription } = this.suiteAdditionalCustomParams;
+
+        return suiteDescription && suiteDescription.filter(text => text.suite === suite).map(item => item.text);
+    }
+
     suiteStarted(suite) {
-        const { attributes, description } = this.additionalCustomParams;
+        const suiteTitle = suite.description;
+
+        if (!(this.suiteAdditionalCustomParams.suiteAttributes && this.suiteAdditionalCustomParams.suiteAttributes.length)) {
+            this.suiteAdditionalCustomParams.suiteAttributes = this.additionalCustomParams.attributes;
+        }
+
+        const { suiteAttributes, suiteDescription } = this.suiteAdditionalCustomParams;
+        const attributes = this.getSuiteAttributesBySuite(suiteTitle);
+        const description = this.getSuiteDescriptionBySuite(suiteTitle);
+
+        this.suiteAdditionalCustomParams.suiteAttributes = suiteAttributes && suiteAttributes.slice(attributes.length);
+        this.suiteAdditionalCustomParams.suiteDescription = suiteDescription && suiteDescription.slice(description.length);
 
         let type = this.getTopLevelType();
         let suiteObj = this.client.startTestItem(Object.assign({
             type,
-            description: suite.description,
+            description: suiteTitle,
             name: suite.fullName
-        }, attributes && { attributes }, description && { description }), this.tempLaunchId, this.getParentId());
+        },
+            attributes && attributes.length && { attributes },
+            description.length && { description: description[0] }), this.tempLaunchId, this.getParentId());
```

Comment:
If I don't make mistake, we decided to overwrite description instead of append or keeping the first setted, don't we?